### PR TITLE
#282: Modified VT data extractor to work without importing Brotli and concurrent 

### LINKED ISSUE #282: Modified VT data extractor to work without importing Brotli and concurrent
### TODO:
- make VT data extractor to work without importing Brotli
- make VT data extractor to work concurrent
- add readme file with sample usage

### PR DESCRIPTION
**Pull request by marcinwrobel1986**
### THIS PR:
- added multiprocessing to vt_data_extractor.py
- made brotli optional
- updated readme file

closes #282

### CODE REVIEW

**Comment by marcinwrobel1986** [comment]
@ppebay @lifflander Tests added coverage shows 57%, but in real it is more than 90%, becasue it is multiprocessing and I can not control the processes, just the output.

**Comment by ppebay** [comment]
@marcinwrobel1986 is it alright for me to review it now?

**Comment by marcinwrobel1986** [comment]
@ppebay yes

File: src/lbaf/Utils/vt_data_extractor.py [hunk 1]
```diff
@@ -45,9 +54,8 @@ def _initial_checks(self):
             self.input_data_dir = os.path.join(project_path, self.input_data_dir)
             print(f"Input data directory: {self.input_data_dir}")
         else:
-            print("Input data directory NOT FOUND!")
             sys.excepthook = exc_handler
-            raise SystemExit(1)
+            raise SystemExit("Input data directory NOT FOUND!")
```
Comment (ppebay) [inline src/lbaf/Utils/vt_data_extractor.py:58]: No exclamation marks 
No all-caps

File: src/lbaf/Utils/vt_data_extractor.py [hunk 2]
```diff
@@ -63,9 +71,9 @@ def _process_input_phases(phases_to_extract: list) -> list:
             elif isinstance(phase, str):
                 phase_list = phase.split('-')
                 if int(phase_list[0]) >= int(phase_list[1]):
-                    print('Phase range wrongly declared!')
+                    print("Phase range wrongly declared!")
                     sys.excepthook = exc_handler
-                    raise SystemExit(1)
+                    raise SystemExit("Phase range wrongly declared!")
```
Comment (ppebay) [inline src/lbaf/Utils/vt_data_extractor.py:76]: Same comment as above

**Review by ppebay — CHANGES_REQUESTED** [review]
Remove all exclamation marks in messages
Put all-caps words in lower case

Thanks

**Comment by marcinwrobel1986** [comment]
@ppebay all done. All checks passed.

**Review by ppebay — APPROVED** [review]
Looking good

**Pull request merged**